src/app.js

```diff
@@ -16,10 +17,20 @@ app.engine(
     extname: "hbs",
     layoutsDir: path.join(__dirname, "views", "layouts"),
     defaultLayout: "main",
-    partialsDir: path.join(__dirname, "views", "partials")
+    // partialsDir: path.join(__dirname, "views", "partials")
   })
 );
-app.use(router)
 
 
+app.get("/", (req, res,next) => {
+  get4painting((err,data)=>{
+    if(err)
+      next(err);
```

Comment:
Where are you handling this? This will break if you don't have a global error middleware.